inc/Admin.php

```diff
@@ -428,6 +437,25 @@ private function get_otter_status(): string {
 		return $status;
 	}
 
+	/**
+	 * Update Otter reference key.
+	 *
+	 * @return void
+	 */
+	public function set_otter_ref() {
+		if ( empty( $_POST['nonce'] ) ) {
+			return;
+		}
+
+		if ( ! wp_verify_nonce( sanitize_text_field( $_POST['nonce'] ), 'neve-fse-set-otter-ref' ) ) {
+			return;
+		}
```

Comment:
NIT. We can group this since they are related to nonce. Something like this:

```php
if (
	empty( $_POST['nonce'] ) ||
	! wp_verify_nonce( sanitize_text_field( $_POST['nonce'] ), 'neve-fse-set-otter-ref' ) 
) {
	return;
}
```